### ISSUE TITLE
"Failed to initialize default value: 'IMAGE_default_che_server'. Environment variable with default value was not found." error when installing nightly Eclipse Che using chectl

### ISSUE THREAD

**Issue by dmytro-ndp** [eclipse-che/che#17359]
### Describe the bug
Installation of Eclipse Che by chectl has failed because of operator error:
> time="2020-07-08T16:38:30Z" level=fatal msg="Failed to initialize default value: 'IMAGE_default_che_server'. Environment variable with default value was not found."

#### chectl logs
```
› Current Kubernetes context: 'minikube'
› Installer type is set to: 'operator'
[12:34:52] Verify Kubernetes API [started]
[12:34:52] Verify Kubernetes API...OK [title changed]
[12:34:52] Verify Kubernetes API...OK [completed]
[12:34:52] 👀  Looking for an already existing Eclipse Che instance [started]
[12:34:52] Verify if Eclipse Che is deployed into namespace "che" [started]
[12:34:53] Verify if Eclipse Che is deployed into namespace "che"...it is not [title changed]
[12:34:53] Verify if Eclipse Che is deployed into namespace "che"...it is not [completed]
[12:34:53] 👀  Looking for an already existing Eclipse Che instance [completed]
[12:34:53] ✈️  Minikube preflight checklist [started]
[12:34:53] Verify if kubectl is installed [started]
[12:34:53] Verify if kubectl is installed [completed]
[12:34:53] Verify if minikube is installed [started]
[12:34:53] Verify if minikube is installed [completed]
[12:34:53] Verify if minikube is running [started]
[12:34:53] Verify if minikube is running [completed]
[12:34:53] Start minikube [started]
[12:34:53] Start minikube [skipped]
[12:34:53] → Minikube is already running.
[12:34:53] Check Kubernetes version [started]
[12:34:53] Check Kubernetes version: Found v1.14.3. [title changed]
[12:34:53] Check Kubernetes version: Found v1.14.3. [completed]
[12:34:53] Verify if minikube ingress addon is enabled [started]
[12:34:53] Verify if minikube ingress addon is enabled [completed]
[12:34:53] Enable minikube ingress addon [started]
[12:34:53] Enable minikube ingress addon [skipped]
[12:34:53] → Ingress addon is already enabled.
[12:34:53] Retrieving minikube IP and domain for ingress URLs [started]
[12:34:53] Retrieving minikube IP and domain for ingress URLs...10.0.103.169.nip.io. [title changed]
[12:34:53] Retrieving minikube IP and domain for ingress URLs...10.0.103.169.nip.io. [completed]
[12:34:53] Checking minikube version [started]
[12:34:53] Checking minikube version... 1.1.1 [title changed]
[12:34:53] Checking minikube version... 1.1.1 [completed]
[12:34:53] Check if cluster accessible [started]
[12:34:53] Check if cluster accessible [skipped]
[12:34:53] ✈️  Minikube preflight checklist [completed]
Eclipse Che logs will be available in '/tmp/chectl-logs/1594226092926'
[12:34:53] Start following logs [started]
[12:34:53] Start following Operator logs [started]
[12:34:53] Start following Operator logs...done [title changed]
[12:34:53] Start following Operator logs...done [completed]
[12:34:53] Start following Eclipse Che logs [started]
[12:34:53] Start following Eclipse Che logs...done [title changed]
[12:34:53] Start following Eclipse Che logs...done [completed]
[12:34:53] Start following Postgres logs [started]
[12:34:53] Start following Postgres logs...done [title changed]
[12:34:53] Start following Postgres logs...done [completed]
[12:34:53] Start following Keycloak logs [started]
[12:34:53] Start following Keycloak logs...done [title changed]
[12:34:53] Start following Keycloak logs...done [completed]
[12:34:53] Start following Plugin registry logs [started]
[12:34:53] Start following Plugin registry logs...done [title changed]
[12:34:53] Start following Plugin registry logs...done [completed]
[12:34:53] Start following Devfile registry logs [started]
[12:34:53] Start following Devfile registry logs...done [title changed]
[12:34:53] Start following Devfile registry logs...done [completed]
[12:34:53] Start following logs [completed]
[12:34:53] Start following events [started]
[12:34:53] Start following namespace events [started]
[12:34:53] Start following namespace events...done [title changed]
[12:34:53] Start following namespace events...done [completed]
[12:34:53] Start following events [completed]
[12:34:53] 🏃‍  Running the Eclipse Che operator [started]
[12:34:53] Copying operator resources [started]
[12:34:53] Copying operator resources...done. [title changed]
[12:34:53] Copying operator resources...done. [completed]
[12:34:53] Create Namespace (che) [started]
[12:34:53] Create Namespace (che)...done. [title changed]
[12:34:53] Create Namespace (che)...done. [completed]
[12:34:53] Create ServiceAccount che-operator in namespace che [started]
[12:34:53] Create ServiceAccount che-operator in namespace che...done. [title changed]
[12:34:53] Create ServiceAccount che-operator in namespace che...done. [completed]
[12:34:53] Create Role che-operator in namespace che [started]
[12:34:53] Create Role che-operator in namespace che...done. [title changed]
[12:34:53] Create Role che-operator in namespace che...done. [completed]
[12:34:53] Create ClusterRole che-operator [started]
[12:34:53] Create ClusterRole che-operator...done. [title changed]
[12:34:53] Create ClusterRole che-operator...done. [completed]
[12:34:53] Create RoleBinding che-operator in namespace che [started]
[12:34:53] Create RoleBinding che-operator in namespace che...done. [title changed]
[12:34:53] Create RoleBinding che-operator in namespace che...done. [completed]
[12:34:53] Create ClusterRoleBinding che-operator [started]
[12:34:53] Create ClusterRoleBinding che-operator...done. [title changed]
[12:34:53] Create ClusterRoleBinding che-operator...done. [completed]
[12:34:53] Create CRD checlusters.org.eclipse.che [started]
[12:34:53] Create CRD checlusters.org.eclipse.che...done. [title changed]
[12:34:53] Create CRD checlusters.org.eclipse.che...done. [completed]
[12:34:53] Waiting 5 seconds for the new Kubernetes resources to get flushed [started]
[12:34:58] Waiting 5 seconds for the new Kubernetes resources to get flushed...done. [title changed]
[12:34:58] Waiting 5 seconds for the new Kubernetes resources to get flushed...done. [completed]
[12:34:58] Create deployment che-operator in namespace che [started]
[12:34:59] Create deployment che-operator in namespace che...done. [title changed]
[12:34:59] Create deployment che-operator in namespace che...done. [completed]
[12:34:59] Create the Custom Resource of type checlusters.org.eclipse.che in the namespace che [started]
[12:34:59] Create the Custom Resource of type checlusters.org.eclipse.che in the namespace che...done. [title changed]
[12:34:59] Create the Custom Resource of type checlusters.org.eclipse.che in the namespace che...done. [completed]
[12:34:59] 🏃‍  Running the Eclipse Che operator [completed]
[12:34:59] ✅  Post installation checklist [started]
[12:34:59] PostgreSQL pod bootstrap [started]
[12:34:59] scheduling [started]
[12:40:03] scheduling [failed]
[12:40:03] → ERR_TIMEOUT: Timeout set to pod wait timeout 300000. podExist: false, currentPhase: undefined
[12:40:03] PostgreSQL pod bootstrap [failed]
[12:40:03] → ERR_TIMEOUT: Timeout set to pod wait timeout 300000. podExist: false, currentPhase: undefined
[12:40:03] ✅  Post installation checklist [failed]
[12:40:03] → ERR_TIMEOUT: Timeout set to pod wait timeout 300000. podExist: false, currentPhase: undefined
 ›   Error: Error: ERR_TIMEOUT: Timeout set to pod wait timeout 300000. 
 ›   podExist: false, currentPhase: undefined
 ›   Installation failed, check logs in '/tmp/chectl-logs/1594226092926'
```

The issue could be related to commit https://github.com/eclipse/che-operator/commit/adfe6980313acb0d26e30ba796a98b3efb784f73

### Che version

 - [ ] latest
 - [x] nightly
 - [ ] other: please specify

### Steps to reproduce
1. Install minikube
2. Download chectl https://github.com/che-incubator/chectl/releases/download/20200708110840/chectl-linux-x64.tar.gz
3. Execute the command:
```
chectl server:start --k8spodreadytimeout=180000 --listr-renderer=verbose --platform=minikube --che-operator-cr-patch-yaml=custom-resource-patch.yaml
```

where **custom-resource-patch.yaml** content is the following:
```
spec:
  server:
    cheImage: 'quay.io/eclipse/che-server'
    cheImageTag: 'nightly'
  auth:
    updateAdminPassword: false
```

### Expected behavior
Eclipse Che has been installed successfully.

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [x] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [x] other: CRW CCI

**Comment by AndrienkoAleksandr** [comment]
Can't reproduce using binary from issue. Maybe old templates was cached. Try to clean up cache `/home/${USER}/.cache/chectl` Maybe you can try to clean up it...

**Comment by dmytro-ndp** [comment]
@AndrienkoAleksandr: the issue has been reproduced on clean jenkins slave.
Please, make sure you are following steps to reproduce described in issue description, and have fresh che-operator:nightly image locally.

**Comment by dmytro-ndp** [comment]
Issue has been fixed after Eclipse Che operator nightly image had been rebuilt.

**Comment by gorshkov-leonid** [comment]
Hello, 
This ticket was closed 4 days ago and Che Operator 7.15.2 released 4 days ago. But I have the same error. Have these changes already in Che Operator 7.15.2 or not yet?

**Comment by tomgeorge** [comment]
I have this same error

```
chectl/7.14.3 linux-x64 node-v10.21.0
```
